spacy/__init__.py

```diff
@@ -14,6 +15,12 @@
 from .errors import Warnings, deprecation_warning
 from . import util
 
+if __version__ >= '2.1.0' and sys.maxunicode <= 65535:
+    raise ValueError('''You are running a narrow unicode build,
```

Comment:
How about `SystemError`? That's the closest I can think of, because it expresses there's something wrong with the system configuration?

Also, we can move the error message text to [`spacy/errors.py`](spacy/errors.py), to keep the texts tidy and in one place. We introduced this a while ago and it made the code so much easier to read, because there wouldn't be big blocks of messages everywhere.

Using the error message would then look something like this:

```python
from .errors import Errors

raise SystemError(Errors.E123)  # or whatever the number is
```